FordLabs/peoplemover
# Move userSpaces state from redux to recoil

## Issue
Continuing the effort to migrate the global state from redux to recoiljs.

## What was done
- [x] Move userSpaces state from redux to recoil
- [x] Move fetching of user spaces from redux to a react hook 

## How to test
1. You should be able to do everything as you did before.
2. Ensure you can go to the dashboard and see your user spaces.
3. Ensure you can log in and see your user name.
4. Ensure you can successfully transfer ownership of a space and see that the dashboard refreshes your "spaces" list to then not include the one that was transferred.
